styles/web/sass/ui_resources/atlas_ui_resources/layouts/_layout-atlas-responsive.scss

```diff
@@ -8,6 +8,7 @@
 
     @media (min-width: $screen-md) {
         .mx-scrollcontainer:not(.mx-scrollcontainer-open) > .region-sidebar {
+            visibility: hidden;
```

Comment:
@wesleysieses the visibility: visible is not being re-set when the sidebar is open..